source/_integrations/bluetooth.markdown

```diff
@@ -19,6 +19,10 @@ The Bluetooth integration will detect nearby Bluetooth devices. Discovered devic
 
 {% include integrations/config_flow.md %}
 
+## Before you begin
+
+The Linux kernel does not prioritize Bluetooth stability and whilst it is possible to use your own Bluetooth card (or one of the many high performance adapter listed below), there may be issues. An alternative "just works" solution is to use an ESP32 and ESPhome's [Bluetooth Proxy](https://esphome.github.io/bluetooth-proxies/). This will provide you with a much smoother experience, espeically if you virtualize and pass thru your adapter. More information is available in the Remote Adapters section below.
```

Comment:
While this is definitely my experience, I think its a bit harsh to put in the documentation.  I'd soften the language a bit here.

Reply:
Yes, I see what you mean. How is [this](https://github.com/home-assistant/home-assistant.io/pull/28944/commits/92256200e70c2300b459f60cb0747aed21188131)?